### DISCUSSION TITLE
Add an option to remove new line and hyphens from header in config

### DISCUSSION THREAD

**Discussion by 123peter123** [CogentRedTester/mpv-file-browser#103] [Ideas]
Hi, I'd love an option to be able to remove the new line and hyphens from the header in the config, instead of manually removing `append('\\N ----------------------------------------------------')` in the modules/ass.lua 

it'd be really nice if we could choose what we append in the config <3 thank you

Example:
![Screenshot from 2024-01-15 18-46-51](https://github.com/CogentRedTester/mpv-file-browser/assets/46938844/cee6c62f-9aec-4ec7-af80-a1d6ea7ea4eb)

and maybe an option to add less new lines between the wrapper too.

**Comment by CogentRedTester** [comment]
The latest commit has added three new config options:

```properties
# The format string used for the header. Uses custom-keybind substitution codes to
# dynamically change the contents of the header. See: docs/custom-keybinds.md#codes
format_string_header=%q\N----------------------------------------------------

# The format strings used for the wrappers. Supports custom-keybind substitution codes, and
# supports two additional codes: `%<` and `%>` to show the number of items before and after the visible list, respectively.
# Setting these options to empty strings will disable the wrappers.
format_string_topwrapper=%< item(s) above\N
format_string_bottomwrapper=\N%> item(s) remaining
```

This allows you to control the contents of the header and wrappers, including newlines (`\N`). This also adds several extra possibilities; the header now supports using the custom-keybinds [substitution codes](https://github.com/CogentRedTester/mpv-file-browser/blob/master/docs/custom-keybinds.md#codes).

For example you could do: `format_string_header=[%i/%x] %q\N----------------------------------------------------`, and you get:
![image](https://github.com/CogentRedTester/mpv-file-browser/assets/57207260/028548ac-bf76-45ce-9672-4a36879df13b)

**Reply by CogentRedTester** [reply]
@123peter123 let me know if you've got them working and if you have any other styling suggestions. I'm not fully satisfied with how the wrappers work/look and I've been considering trying to customise them.

**Reply by 123peter123** [reply]
I'm happy with it, but I don't know much about development. I really appreciate you using my suggestion though ❤️

The only styling edit I made for the wrappers in the config was remove the new lines (\N) so I could have 25 items on the screen instead of 24 without the wrapper going off the screen.
